astropy/units/quantity.py

```diff
@@ -114,6 +115,16 @@ def __next__(self):
     next = __next__
 
 
+class QuantityColumnInfo(ColumnInfo):
+    _attrs_from_parent = set(['dtype', 'unit'])
+
+    def __init__(self, *args, **kwargs):
+        super(QuantityColumnInfo, self).__init__(*args, **kwargs)
+
+    def default_format_func(self, format, val):
```

Comment:
I wonder whether it wouldn't be clearer conceptually to just call this `default_format` and let it have exactly the same meaning as `Column.format`, i.e., if it is callable, it is a function, and if not, it is tried to interpret it as a format string.  I realise this makes it slightly slower to print, but I think it is worth the clarity.  With that and the comment above, one would just have
```
class QuantityColumnInfo(ColumnInfo):
    _attrs_from_parent = set(['dtype', 'unit'])
    default_format = '{0.value}'
```